app/views/layouts/application.html.erb

```diff
@@ -2,15 +2,17 @@
 <html>
   <head>
     <title>MyBlog</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
```

Comment:
- [ ] Please fix this error currently I am not able to run the server
![tal](https://user-images.githubusercontent.com/44745855/155544653-cd405539-15d6-43f5-a73a-3a66410930d7.png)